TFG-Labs/custom-shipping-step-by-items
# Tlva 57 - Mixed Products

### What problem is this solving?

Functionality developed to satisfy customer requirements for mixed products to display an alert for the custommer.

#### How it works

If there are products with furniture category and another category in the shopping cart, the customer will be shown an alert.

#### How to test it?

[Workspace](https://tlva57--thefoschini.myvtex.com/)

You must be logged in with a customer already registered on the platform.

In the checkout add the product with SKU **59418103 and 59418159**, which one has category "furniture" and the other product a different category.

When you reach the Shipping step and you have entered the shipping address, the alert will appear as you can see in the following video.

### Screenshots/Video example usage

[Video](https://www.loom.com/share/3847203b62714a3a859bf667eaefd73d)

#### How does this PR make you feel? 🔗

![](https://media.giphy.com/media/lNrNLRLmpC3VIjl82D/giphy.gif)
